Antenna_Performance_Metric/count_jobs.py

```diff
@@ -0,0 +1,34 @@
+# Count the number of jobs from string of format:
+# 28178971_1 28178971_2 28178971_[3-50]
+import argparse
+import re
+
+def expand_array_job(job_id):
+    '''expand 28178971_[3-50] to 28178971_3, 28178971_4, ..., 28178971_50'''
+    if re.match(r'\d+_\[\d+-\d+\]$', job_id):
+        prefix, range_str = job_id.split('_')
+        start, end = map(int, range_str[1:-1].split('-'))
+        return [f"{prefix}_{i}" for i in range(start, end + 1)]
+    else:
+        return [job_id]
+
+
+def read_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('job_ids', type=str, help='Count the number of jobs from the squeue string')
+    args = parser.parse_args()
+    return args
+
+
+def main(args):
+    '''Count the number of jobs from the squeue string'''
+    jobs = args.job_ids.split()
+    job_count = 0
+    for job in jobs:
+        job_count += len(expand_array_job(job))
```

Comment:
If you are only using `expand_array_job` to get the number of jobs you could just write a function that extracts the start and end and returns the difference, but if you are planning to reuse the expanded array elsewhere the function you have now makes sense.

Reply:
Yeah, originally I was going to use it for something else, but I ended up not. I just added another function to count the number from start and end, leaving the expanded array function in case I need it in the future